src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -369,6 +370,8 @@ ensogl::def_command_api! { Commands
     /// Switches the selected visualisation to/from fullscreen mode.
     toggle_fullscreen_for_selected_visualization,
 
+    /// Simulates a style toggle press event.
+    toggle_style,
```

Comment:
this should not be a part of GraphEditor API, because you switch styles in the whole application.

On main there is "project::View" class, please move the logic there.